mail_footer_notified_partner/models/mail_followers.py

```diff
@@ -9,9 +9,29 @@
 class MailNotification(models.Model):
     _inherit = 'res.partner'
 
+    @api.multi
+    def _notify_by_email(
+        self, message, force_send=False, send_after_commit=True,
+        user_signature=True
+    ):
+        # we need to save the complete list of partners because
+        # _message_notification_recipients builds recipients
+        # grouped by users groups. Thus get_additional_footer would get a
+        # partial list of recipients
+        ctx = dict(self.env.context)
+        ctx.update({"notified_partners": self})
+        return super(
+            MailNotification, self.with_context(ctx))._notify_by_email(
```

Comment:
how about 

```python
return super(
    MailNotification, self.with_context(notified_partners=self)
)._notify_by_email(
    ...
)
```

Reply:
@hbrunn this would produce a context with ```notified_partners``` only, right? In this case we need to keep the other items of original self.env.context

Reply:
https://github.com/OCA/OCB/blob/10.0/odoo/models.py#L4867

Reply:
@hbrunn thanks! fixed